src/components/Search/index.stories.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import Search from '.';
+import '../../styles/stories.scss';
+import Button from '../Button';
+
+export default {
+  title: 'Search',
+  component: Search
+};
+
+export const BasicUsage = () => (
+  <div className='container column grey'>
+    <Search />
+    <Search shape='round' />
+  </div>
+);
+
+export const CustomLabel = () => (
+  <div className='container column grey'>
+    <Search
+      placeholder='Label width of 65px'
+      label='Address'
+      labelWidth='65px'
+    />
+    <Search label='Address' labelAlign='center' />
+    <Search label='Address' labelAlign='right' />
+  </div>
+);
+
+export const BackgroundColor = () => (
+  <div className='container grey'>
+    <Search background='rgb(79, 192, 141)' />
+  </div>
+);
+
+export const MaxLength = () => (
+  <div className='container grey'>
+    <Search maxLength={5} />
+  </div>
+);
+
+export const PlaceholderAutoFocus = () => (
+  <div className='container grey column'>
+    <Search placeholder='This is a placeholder' />
+    <Search placeholder='This is a auto focus' autofocus />
+  </div>
+);
+
+export const SearchActions = () => {
+  const handleClick = (e) => {
+    e.preventDefault();
+    alert('Action clicked');
+  };
+  const [value, setValue] = useState('');
+  const [focus, setFocus] = useState(false);
+  return (
+    <div className='container column grey'>
+      <h1>Value: {value}</h1>
+      <Search cancel={handleClick} showAction />
+      <Search cancel={handleClick} showAction actionText='Clear' />
```

Comment:
![Screen Shot 2020-05-28 at 8 11 47 PM](https://user-images.githubusercontent.com/59517542/83206751-83789f80-a11f-11ea-9e48-c20eeb04e3a0.png)
out line liiks bad

Reply:
and cancel doesnt work here